### PR TITLE
Add Scheme

### PR DESCRIPTION
**Pull request by Retia-Adolf**
[theodesp/awesome-scheme](https://github.com/theodesp/awesome-scheme)

**Scheme is an awesome programming language, a branch of Lisp family, which has the simplest and the most beautiful syntax based on S-expressions, parenthesized lists with the distinct and powerful polish notation.**

**Scheme follows a minimalist design philosophy specifying a small standard core with powerful tools for language extension. It's the first dialect of Lisp to choose lexical scope and the first to require implementations to perform tail-call optimization, as well as first full production system to support first-class continuations. Also generated an offspring language called Racket.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- [ ] **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- [ ] I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- [x] The entry in the Awesome list should:
	- [x] Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- [x] Be added at the bottom of the appropriate category.
- [ ] The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [ ] It's the result of hard work and the best I could possibly produce.
- [ ] Non-generated Markdown file in a GitHub repo.
- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- [ ] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- [x] Not a duplicate.
- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
- [ ] Includes a project logo/illustration whenever possible.
	- [ ] Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- [ ] The image should link to the project website or any relevant website.
	- [ ] The image should be high-DPI. Set it to maximum half the width of the original image.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- [x] Should be placed on the right side of the readme heading.
	- [x] Should link back to this list.
- [x] Has a Table of Contents section.
	- [x] Should be named `Contents`, not `Table of Contents`.
	- [x] Should be the first section in the list.
	- [x] Should only have one level of sub-lists, preferably none.
- [x] Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- [x] That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [x] [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- [x] If you use a license badge, it should be SVG, not PNG.
- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- [x] The file should be named `contributing.md`. Casing is up to you.
- [x] Has consistent formatting and proper spelling/grammar.
	- [x] The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- [x] The description starts with an uppercase character and ends with a period.
	- [x] Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by theodesp** [comment]
I'm more than happy to add this to https://github.com/sindresorhus/awesome but to be honest the Requirements criteria scared me the first time and so used this  generator https://github.com/dar5hak/generator-awesome-list

**Comment by uraimo** [comment]
Nice list, Scheme rocks.
This looks good to me after the patch from Retia, the only thing missing is a real `contributing.md` that at the moment has a lot of "take care of this and that".
You could take inspiration from one of the list already accepted like the one on [nodejs](https://github.com/theodesp/awesome-scheme/blob/master/contributing.md), adding typographical rules should be more or less enough in most cases (and should save you a lot of manual work when someone submits something).

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/theodesp/awesome-scheme
✖ Linting

  README.md:21:3
  ✖   1:19  Invalid badge source                                                   remark-lint:awesome/badge
  ✖  21:3   Link to https://mitpress.mit.edu/sites/default/files/6515.pdf is dead  remark-lint:no-dead-urls

  2 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 1]
```diff
@@ -173,6 +173,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3#readme) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta#readme) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris#readme) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [Scheme](https://github.com/theodesp/awesome-scheme#readme) - Minimalistic prophet and pioneer programming language to be with lexical scope, lambda caculus, block structure, proper tail recursion, and first class continuations.
```
Comment (sindresorhus) [inline readme.md]: Can you try to shorten the description a little bit?

**Comment by sindresorhus** [comment]
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

**Comment by sindresorhus** [comment]
I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community

**Comment by sindresorhus** [comment]
Remove unused content instead of commenting it out:

```
<!-- trick for github not render --><!-- - [An Introduction to Scheme and its Implementation] - ftp://ftp.cs.utexas.edu/pub/garbage/cs345/schintro-v14/schintro_toc.html - Learn and use Scheme. Great Resource. -->
```

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**